internal/config/config.go

```diff
@@ -53,6 +55,17 @@ type AppConfig struct {
 	ArchivePath string
 }
 
+type SessionConfig struct {
+	// Session key expiration time.
```

Comment:
i think we should disable it by default, if the `Session_expiration` and `Session_domain` are not set

e.g. lines 126-128 or 218-220
```
if viper.IsSet("configFile") {
		viper.SetConfigFile(viper.GetString("configFile"))
}
```

Reply:
sessions disabled by default, and also fixed a time-to-int conversion bug that this made apparent (`time.Duration` is in nanoseconds, but cookie age is in seconds, so cookie lifetime got accidentally increased by a factor of `1e9`)